pilot/pkg/config/kube/crd/conversion.go

```diff
@@ -78,10 +95,14 @@ func ConvertObject(schema collection.Schema, object IstioObject, domain string)
 	if err != nil {
 		return nil, err
 	}
-	data, err := FromJSON(schema, string(js))
+	spec, err := FromJSON(schema, string(js))
 	if err != nil {
 		return nil, err
 	}
+	status, err := IstioStatusJSONFromMap(object.GetStatus())
+	if err != nil {
+		log.Errorf("could not get istio status from map %v, err %v", object.GetStatus(), err)
```

Comment:
Any reason to log the error here but the previous one we will return an error?

Reply:
i dont think we want status breaking/erroring if something goes wrong, right? could be wrong